src/feed/tests/views/test_funding_status_priority_sorting.py

```diff
@@ -0,0 +1,123 @@
+        self.client = APIClient()
+        self.client.force_authenticate(user=self.user)
+        self.now = timezone.now()
+        self.counter = 0
+        
+        RscExchangeRate.objects.create(
+            price_source=MORALIS, rate=3.0, real_rate=3.0, target_currency=USD
+        )
+
+    def _create_fundraise(self, status, days_offset):
+        self.counter += 1
+        doc = ResearchhubUnifiedDocument.objects.create(document_type=PREREGISTRATION)
+        post = ResearchhubPost.objects.create(
+            title=f"Post {self.counter}",
+            created_by=self.user,
+            document_type=PREREGISTRATION,
+            unified_document=doc,
+            renderable_text="Test",
+            slug=f"test-{self.counter}",
+        )
+        escrow = Escrow.objects.create(
+            created_by=self.user,
+            hold_type=Escrow.FUNDRAISE,
+            amount_holding=100,
+            content_type=ContentType.objects.get_for_model(ResearchhubUnifiedDocument),
+            object_id=doc.id,
+        )
+        Fundraise.objects.create(
+            created_by=self.user,
+            unified_document=doc,
+            status=status,
+            end_date=self.now + timedelta(days=days_offset),
+            escrow=escrow,
+        )
+        return post
+
+    def _create_grant(self, status, days_offset):
+        self.counter += 1
+        doc = ResearchhubUnifiedDocument.objects.create(document_type=GRANT)
+        post = ResearchhubPost.objects.create(
+            title=f"Grant {self.counter}",
+            created_by=self.user,
+            document_type=GRANT,
+            unified_document=doc,
+            renderable_text="Test",
+            slug=f"grant-{self.counter}",
+        )
+        Grant.objects.create(
+            created_by=self.user,
+            unified_document=doc,
+            status=status,
+            amount=10000,
+            description="Test",
+            end_date=self.now + timedelta(days=days_offset),
+        )
+        return post
+
+    def _get_ids(self, response):
+        return [item["content_object"]["id"] for item in response.data["results"]]
+
+    def test_funding_feed_prioritizes_open_status(self):
+        open_post = self._create_fundraise(Fundraise.OPEN, 10)
+        closed_post = self._create_fundraise(Fundraise.COMPLETED, -5)
+        
+        response = self.client.get(reverse("funding_feed-list"))
+        ids = self._get_ids(response)
+        
+        self.assertLess(ids.index(open_post.id), ids.index(closed_post.id))
+
+    def test_grant_feed_prioritizes_open_status(self):
+        open_grant = self._create_grant(Grant.OPEN, 10)
+        closed_grant = self._create_grant(Grant.CLOSED, -5)
+        
+        response = self.client.get(reverse("grant_feed-list"))
+        ids = self._get_ids(response)
+        
+        self.assertLess(ids.index(open_grant.id), ids.index(closed_grant.id))
+
+    def test_custom_ordering_maintains_status_priority(self):
```

Comment:
test name is confusing. What are we doing here?
The words: "custom ordering" and "status priority" are not understandable at a glance. No clue what they are doing without having to read and understand the code

Good test name would be `test_show_open_fundraises_before_closed`, etc...